<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt-symlinktagsfs</string>
  </property>
  <widget class="QLabel" name="labelyuanmulu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>源目录   ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelmubiaomulu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>目标目录：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdityuanmulu">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>541</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditmubiaomulu">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>541</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonchyuanmulu">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>选择源目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonchmubiaomulu">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>95</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>选择目标目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncreate">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDelete">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>520</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除歌曲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonChIDV3">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改IDV3标签</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Title :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelArtist">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAlbum">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> Album:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelYear">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Year :</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComment">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Comment :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTitle">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditArtist">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAlbum">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>260</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>330</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetIDV3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>获取IDV3标签</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonrefresh">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncopy">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
